### PR TITLE
QAE-350 - E2E test case for disable breadcrumb on various pages for after header position

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Added test case for disable breadcrumb on various pages for after header position

### Screenshots

https://share.getcloudapp.com/d5uNe5zK

### How has this been tested?

Run single test: npm run test:e2e:interactive tests/e2e/specs/customizer/breadcrumbs/after-disable-breadcrumb-on-various-pages.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/breadcrumbs/after-disable-breadcrumb-on-various-pages.test.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	// 	await expect( enableSearchPage ).toBeNull( );
+	// } );
+
+	it( 'disable breadcrumb on archive page after header position should apply corectly', async () => {
+		const insideBreadcrumb = {
+			'breadcrumb-position': 'astra_header_after',
+			'breadcrumb-disable-archive': 1,
+		};
+		await setCustomize( insideBreadcrumb );
+		await page.goto( createURL( '/author/admin' ), {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/breadcrumbs/after-disable-breadcrumb-on-various-pages.test.js:61]: test case is failing as there is not such url "/author/admin"

File: tests/e2e/specs/customizer/breadcrumbs/after-disable-breadcrumb-on-various-pages.test.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
+import { createURL, createNewPost } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../utils/customize';
+import { publishPost } from '../../../utils/publish-post';
+describe( 'breadcrumb settings in the customizer', () => {
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/breadcrumbs/after-disable-breadcrumb-on-various-pages.test.js:4]: First letter capital

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/breadcrumbs/after-disable-breadcrumb-on-various-pages.test.js]: Done

**Review by bsfsnehal — APPROVED** [review]

File: tests/e2e/specs/customizer/breadcrumbs/after-disable-breadcrumb-on-various-pages.test.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+		await setCustomize( afterBreadcrumb );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-primary-header-bar' );
+		const enablehomePage = await page.$eval( '.ast-primary-header-bar', ( element ) => element.getAttribute( '.site-header-focus-item + .ast-breadcrumbs-wrapper' ) );
+		await expect( enablehomePage ).toBeNull( );
+	} );
+
+	it( 'disable breadcrumb on blog page for after header position should apply corectly', async () => {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/breadcrumbs/after-disable-breadcrumb-on-various-pages.test.js:19]:
```suggestion
	it( 'disable breadcrumb on blog page for after header position should apply correctly', async () => {
```

Comment (bsfsnehal) [reply tests/e2e/specs/customizer/breadcrumbs/after-disable-breadcrumb-on-various-pages.test.js:19]: same for other sub test cases

File: tests/e2e/specs/customizer/breadcrumbs/after-disable-breadcrumb-on-various-pages.test.js [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+	} );
+
+	it( 'disable breadcrumb on search page after header position should apply correctly', async () => {
+		const afterBreadcrumb = {
+			'breadcrumb-position': 'astra_header_after',
+			'breadcrumb-disable-archive': 1,
+		};
+		await setCustomize( afterBreadcrumb );
+		const ppStatus = false;
+		while ( false === ppStatus ) {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/breadcrumbs/after-disable-breadcrumb-on-various-pages.test.js:45]: @SimranChougule1 this test case is failing at local, can you please check this?

Comment (bsfsnehal) [reply tests/e2e/specs/customizer/breadcrumbs/after-disable-breadcrumb-on-various-pages.test.js:45]: @SimranChougule1 please execute this test case at local-end

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/breadcrumbs/after-disable-breadcrumb-on-various-pages.test.js:45]: Resolved

**Review by bsfsnehal — APPROVED** [review]

**Review by GauravKhupse — APPROVED** [review]

**Pull request merged**